reports/GCP-e2-highmem-16.md

```diff
@@ -0,0 +1,85 @@
+OS: Ubuntu 22.04.3 LTS x86_64  
+Host: Google Compute Engine  
+Kernel: 6.2.0-1014-gcp  
+CPU: AMD EPYC 7B12 (16) @ 2.249GHz  
+Memory: 502MiB / 128802MiB  
```

Comment:
What is this? AFAICS e2-highmem-16 have 128GiB of RAM what is 502MiB? LLC? Unlikely according to specs it is 256MiB.

Reply:
This is the memory being used at that moment.